src/main/java/org/openstreetmap/atlas/geography/atlas/raw/sectioning/WaySectionChangeSet.java

```diff
@@ -129,4 +131,10 @@ public void recordPoint(final Point point)
     {
         this.pointsToStayPoints.add(point.getIdentifier());
     }
+
+    private Set<TemporaryNode> getCreatedNodes()
+    {
+        return this.edgeToNodeMapping.values().stream()
```

Comment:
If that is not reused, why not using the initial stream line 108 so it does not have to be collected and re-streamed?

Reply:
Good point, this doesn't even need to exist.